### PR TITLE
Add stm32p103 emulator instruction

### PR DESCRIPTION
**Pull request by mlouielu**
Add instruction for how to run stm32p103 emulator.

### CODE REVIEW

**Comment by jserv** [comment]
Let's move the instructions into a new dedicated page at directory "Documentation/".

**Comment by mlouielu** [comment]
Do you want to use sphinx, or plain text is enough?

**Comment by jserv** [comment]
Using plain text should work.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,135 @@
...
+Creating /proc/version
+Creating /proc/meminfo
+Creating /dev/mem
+Creating /dev/null
+Creating /dev/zero
+Creating /dev/random
+Creating MTD device mtd0
+Kernel bootstrap done.
+--
+Piko version 0.0.0 (uscc@uscc-Inspiron-7559) (armv7-m, cortex-m3) #Sun Apr 30 17:58:03 2017
```
Comment (jserv) [inline README.md]: The two lines were generated by build host, and you don't have to include explicitly.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,135 @@
...
+$ make -j8
+```
+
+Make sure you have export the `arm-softmmu` to `$PATH`
+
+```
+$ export PATH=$PATH:~/qemu_stm32-stm32_v0.1.3/arm-softmmu
+```
+
+You will need to modify the target from `stm32f429` to
```
Comment (jserv) [inline README.md]: It is not necessary to modify `Makefile`. Instead, you only have to execute `make TARGET=stm32p103`.

**Comment by mlouielu** [comment]
Close this by proposing #13

**Pull request closed without merging**